code/modules/uplink/uplink_items.dm

```diff
@@ -233,7 +233,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A dusty SUPER-SIZED from the back of the Syndicate warehouse. Rumored to contain a valuable assortment of items, \
 			but you never know. Contents are sorted to always be worth 125 TC."
 	cost = 50 //SKYRAT EDIT CHANGE - ORIGINAL: 40
-	player_minimum = 40
+	player_minimum = 80 //SKYRAT EDIT CHANGE - ORIGINAL: 40 - We never have less than 50 people.
```

Comment:
🤡 

no.

Reply:
We wanna discourage traitors teaming up anyway.